lookout/style/format/analyzer.py

```diff
@@ -313,20 +333,21 @@ def _generate_file_comments(
                 fix_vnodes.append(line_vnode_y)
                 fix_ys.append(line_y_predi)
             confidence = self._get_comment_confidence(line_ys, line_ys_pred, line_winners, rules)
-            text = self.render_comment_text(
+            yield dict(
+                ok=True,                      # generation success
                 language=language,            # programming language of the code
                 line_number=line_number,      # line number for the comment
-                code_lines=code_lines,        # original file code lines
+                head_file=file,               # file from head revision
+                base_file=base_file,          # file from base revision
                 new_code_line=new_code_line,  # code line suggested by our model
-                winners=fix_rules,            # winner rule indices
-                vnodes=fix_vnodes,            # corrected VirtualNode-s
-                fixed_labels=fix_ys,          # predicted labels for those nodes
+                vnodes=fix_vnodes,            # VirtualNode-s of the line where changes were
+                                              # introduced.
+                winners=fix_rules,            # winner rule indices for the vnodes
+                fixed_labels=fix_ys,          # new predicted labels for those nodes
                 confidence=confidence,        # overall confidence in the prediction, 0-100
-                feature_extractor=fe,         # FeatureExtractor involved.
+                feature_extractor=fe,         # FeatureExtractor involved
+                new_vnodes=new_vnodes,        # fixed vnodes via CodeGenerator for all file
```

Comment:
We agreed that `new_vnodes` covers `fix_vnodes` and `fix_ys`, but it is not so easy to exclude it and requires more code changes, so I keep it as is for now. 

I can do it now if you want.